kratos/utilities/multipoint_constraint.h

```diff
@@ -0,0 +1,590 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|

```

Comment:
I think this class is missing the serializer functtions

Reply:
Added .. !